### PR TITLE
Core Setup

### PR DESCRIPTION
**Pull request by PaulPilane**
# Description

Angular core module and routing setup for project, along with the addition of ionic libraries.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Angular app passes all predefined library tests.

- [x] Nx Welcome Test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/app/src/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </head>
 
 <body>
-    <fridge-to-plate-root></fridge-to-plate-root>
+    <core-shell></core-shell>
```
Comment (kphutiyagae) [inline apps/app/src/index.html:13]: Why did we switch from AppModule to CoreModule?

Comment (ryanbasiltrickett) [reply apps/app/src/index.html:13]: This is just a renaming of the core module component

**Review by Shojiki-Lukhozi — DISMISSED** [review]

**Review by kphutiyagae — CHANGES_REQUESTED** [review]
Please address comments before merge.

**Review by ryanbasiltrickett — DISMISSED** [review]

**Review by kphutiyagae — APPROVED** [review]
Looks good, approved.

**Review by ryanbasiltrickett — APPROVED** [review]

**Pull request merged**